Framework/include/QualityControl/QcInfoLogger.h

```diff
@@ -27,6 +27,14 @@ typedef AliceO2::InfoLogger::InfoLoggerContext infoContext;
 namespace o2::quality_control::core
 {
 
+struct DiscardFileParameters {
+  bool debug = false;
+  int fromLevel = 21 /* Discard Trace */;
+  std::string discardFile;
+  unsigned long rotateMaxBytes = 0;
+  unsigned int rotateMaxFiles = 0;
+};
+
```

Comment:
as a future improvement, I would propose to move this structure to a separate file, so we do not include QcInfologger header in configuration structures.

Reply:
agreed, will do